db/dcrpg/pgblockchain.go

```diff
@@ -4827,6 +4827,19 @@ func (pgb *ChainDB) GetAPITransaction(txid *chainhash.Hash) *apitypes.Tx {
 		return nil
 	}
 
+	msgTx, err := txhelpers.MsgTxFromHex(txraw.Hex)
+	if err != nil {
+		log.Errorf("Cannot create MsgTx for tx %v: %v", txid.String(), err)
+		return nil
+	}
+
+	treasuryActive := true
```

Comment:
In the future, the intent should be documented. `// default for mempool transactions`